specifyweb/frontend/js_src/lib/components/QueryBuilder/FieldFilterTool.tsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+
+import { commonText } from '../../localization/common';
+import { queryText } from '../../localization/query';
+import type { RA } from '../../utils/types';
+import { Button } from '../Atoms/Button';
+import { className } from '../Atoms/className';
+import { iconClassName, icons } from '../Atoms/Icons';
+import {
+  mappingElementDivider,
+  mappingElementDividerClassName,
+} from '../WbPlanView/LineComponents';
+import type { QueryFieldFilter } from './FieldFilter';
+import type { QueryField } from './helpers';
+export function FieldFilterTool({
+  fieldFilters,
+  index,
+  isBasic,
+  hasAny,
+  isFieldComplete,
+  fieldName,
+  handleChange,
+  handleFilterChange,
+}: {
+  readonly fieldFilters: RA<{
+    readonly type: QueryFieldFilter;
+    readonly startValue: string;
+    readonly isNot: boolean;
+    readonly isStrict: boolean;
+  }>;
```

Comment:
(optional) could you extract the following into a separate typescript type since it's getting larger?

```ts
{
    readonly type: QueryFieldFilter;
    readonly startValue: string;
    readonly isNot: boolean;
    readonly isStrict: boolean;
  }
```

Reply:
then, would it be appropriate to add a comment on top of `isStrict` to explain briefly what this is responsible for?